docs/network/architecture.md

```diff
@@ -13,12 +13,27 @@ These components are instances of binaries in the [Octez software suite](https:/
 ## Sequencer
 
 Etherlink relies on a sequencer to publish transactions.
-It receives transactions from EVM nodes, puts them in order, and publishes them in two ways:
+It receives transactions from EVM nodes, puts them in order, and packages them into a blueprint, which is a group of Etherlink transactions.
 
-- It publishes transactions to EVM nodes, which can consider them final as long as they trust that the sequencer will publish them to layer 1.
-- It publishes transactions to Tezos layer 1, which finalizes them.
+Each blueprint contains:
+
+- A list of transactions
+- A list of transactions that are currently in the delayed inbox, as described in [Transaction lifecycle](#transaction-lifecycle)
+- The hash of the previous blueprint
+- The timestamp of the blueprint.
+
+You can imagine blueprints as the blocks in Etherlink's layer 2 chain.
```

Comment:
Not sure about this, but I'm not sure yet what to propose instead

Reply:
This could be confusing -- removed it.